test/testiconutils.cpp

```diff
@@ -29,12 +29,56 @@ class TestIconUtils : public QObject
     }
 
 private slots:
+    void testDrawSvgWithCustomFillColor()
+    {
+        const QString blackSvgDirPath{QString{OCC::Theme::themePrefix} + QStringLiteral("black")};
+        const QDir blackSvgDir(blackSvgDirPath);
+        const QStringList blackImages = blackSvgDir.entryList(QStringList("*.svg"));
+
+        if (!blackImages.isEmpty()) {
```

Comment:
@mgallien How can we do that? Should I provide this test with programmatically-generated SVG, or, add .svg files just for this unit test? Otherwise, we always have some images in the `OCC::Theme::themePrefix/black`.

Reply:
I do not really understand why the test can be no-op in case you have no images
is there a situation where we would run the tests but `blackImages.isEmpty()` is true ?
I would just `Q_ASSERT(!blackImages.isEmpty())`